src/Profile.php

```diff
@@ -66,6 +66,8 @@ public static function fromUuid($uuid, ApiClient $apiClient = null)
             }
         }
 
+        $profile->history = $apiClient->uuidHistory($uuid);
```

Comment:
I like where you were going with this, but due to Mojang API limitations making the call immediately is a bad decision.  Their rate-limits are incredibly touchy.  Reaching out preemptively like this also drastically increases script execution time.

I think the best way of handling this is to add a `getHistory()` instance method to the `Profile` class.  It would look something like:

    public function getHistory(ApiClient $apiClient = null)
    {
        if (is_null($apiClient)) {
            $apiClient = static::createApiClient();
        }
        if (is_null($this->history)) {
            $this->history = $apiClient->uuidHistory($this->uuid);
        }
        return $this->history;
    }

If you wanted to provide this same functionality still, I would also suggest creating a new static method:

    public static function fromUuidWithHistory($uuid, ApiClient $apiClient = null)
    {
        if (is_null($apiClient)) {
            $apiClient = static::createApiClient();
        }
        $profile = static::fromUuid($uuid, $apiClient);
        $profile->getHistory(); // Eagerly load History
        return $profile;
    }